istioctl/cmd/root.go

```diff
@@ -199,9 +199,8 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(precheck.Cmd(ctx))
 	experimentalCmd.AddCommand(proxyconfig.StatsConfigCmd(ctx))
 	experimentalCmd.AddCommand(checkinject.Cmd(ctx))
-	// TODO: revisit once ambient mode is supported in Sail Operator.
-	rootCmd.AddCommand(unsupportedCmd(waypoint.Cmd(ctx), "none"))
-	rootCmd.AddCommand(unsupportedCmd(ztunnelconfig.ZtunnelConfig(ctx), "none"))
+	rootCmd.AddCommand(waypoint.Cmd(ctx))
```

Comment:
LGTM. 

I'm adding do-not-merge label so that @dgn @longmuir can take a look.

Reply:
I see that Daniel is [okay with enabling](https://github.com/openshift-service-mesh/sail-operator/pull/171#issuecomment-2610407006) the commands.